README.md

```diff
@@ -173,11 +173,15 @@ grunt watch
 Run the unit tests with a local server that supports PHP. Ensure that you run the site from the root directory, not the "test" directory. No database is required. Pre-configured php local servers are available for Windows and Mac. Here are some options:
 
 - Windows: [WAMP download](http://www.wampserver.com/en/)
-- Mac: [MAMP download](http://www.mamp.info/en/index.html)
 - Linux: [Setting up LAMP](https://www.linux.com/learn/tutorials/288158-easy-lamp-server-installation)
 - [Mongoose (most platforms)](http://code.google.com/p/mongoose/)
+- Mac: [MAMP download](http://www.mamp.info/en/index.html)
 
+If you already have PHP installed, you can use it's built in webserver to complete the tests:
 
+```
+php -S localhost:8080
```

Comment:
I've checked locally but such a server doesn't seem to work well. It failed the "core: document ready when jQuery loaded asynchronously (#13655)" test and times out/fails a lot of AJAX tests. I guess a full Apache server might be needed to make it work correctly.